packages/cozy-mespapiers-lib/README.md

```diff
@@ -126,6 +126,20 @@ const MesPapiersView = props => {
 
 export default MesPapiersView
 ```
+# Call modal with URL
+In your application, if you want to call a modal to create a Paper, you just have to call the `/paper/create` or `/paper/create/<qualificationLabel>` route with the query parameter `backgroundPath=<currentPath>` (so that the background in Desktop mode is your current page)
```

Comment:
on peut peut-être écrire le `<qualificationLabel>` au format hashrouter à savoir `/paper/create/:qualificationLabel` ?

Reply:
je supprimerai la phrase `(so that the background in Desktop mode is your current page)` qui n'a pas de lien direct, et apporte de la confusion

Reply:
En effet, modifié 👍